Tests/GitHubRepositories.tests.ps1

```diff
@@ -39,6 +137,84 @@ try
             }
         }
     }
+
+    Describe 'Updating repositories' {
+
+        Context -Name 'For creating a repository' -Fixture {
+            BeforeAll {
+                $repo = New-GitHubRepository -RepositoryName $defaultRepoName -Description $defaultRepoDesc -AutoInit
+            }
+            AfterAll {
+                Remove-GitHubRepository -Uri "$($repo.svn_url)" -Verbose
+            }
+
+            It 'Should have the new updated description' {
+                $updatedRepo = Update-GitHubRepository -RepositoryName $defaultRepoName -Description $modifiedRepoDesc
+                $updatedRepo.description | Should be $modifiedRepoDesc
+            }
+
+            It 'Should have the new updated homepage url' {
+                $updatedRepo = Update-GitHubRepository -RepositoryName $defaultRepoName -Homepage $defaultRepoHomePage
+                $repo.homepage | Should be $defaultRepoHomePage
+            }
+        }
+    }
+
+    Describe 'Get/set repository topic' {
+
+        Context -Name 'For creating and getting a repository topic' -Fixture {
+            BeforeAll {
+                $repo = New-GitHubRepository -RepositoryName $defaultRepoName -AutoInit
+            }
+            AfterAll {
+                Remove-GitHubRepository -Uri "$($repo.svn_url)" -Verbose
+            }
+
+            It 'Should have the expected topic' {
+                $topic = Set-GitHubRepositoryTopic -RepositoryName $defaultRepoName -Name ($defaultRepoTopic)
+                $updatedRepo.names[0] | Should be $defaultRepoTopics
```

Comment:
Isn't that supposed to be `$defaultRepoTopic`? (You added an `s`)

Reply:
Fixed